config/firebaseConfig.ts

```diff
@@ -0,0 +1,18 @@
+import { initializeApp } from "firebase/app";
+import { getAuth } from "firebase/auth";
+
+const firebaseConfig = {
+  apiKey: "AIzaSyC5utU_IjuPt9xSfYICDrHmQywBCZYetVE",
```

Comment:
API Key is exposed! make use of `.env` variables

Reply:
So, as per the firebase official docs, it is safe to expose the firebase API key.
I'll still use env variables, for good practices.